src/query/members.ts

```diff
@@ -74,20 +74,32 @@ export const useUpdateMember = () => {
 
 export const useDeleteMember = () => {
   const queryClient = useQueryClient();
+  const openSnackBar = useSnackBar();
   return useMutation({
     mutationFn: (id: number) => deleteMember(id),
     onSuccess: () => {
       queryClient.invalidateQueries({ queryKey: ['members'] });
+      openSnackBar({ type: 'success', message: '회원 삭제에 성공했습니다.' });
+    },
+    onError: (e) => {
+      const err = e as AxiosError;
+      openSnackBar({ type: 'error', message: (err.response?.data as ErrorResponse).message });
     },
   });
 };
 
 export const useCreateMember = () => {
   const queryClient = useQueryClient();
+  const openSnackBar = useSnackBar();
   return useMutation({
     mutationFn: (member: MemberCreate) => createMember(member),
     onSuccess: () => {
       queryClient.invalidateQueries({ queryKey: ['members'] });
+      openSnackBar({ type: 'success', message: '회원 추가에 성공했습니다.' });
+    },
+    onError: (e) => {
+      const err = e as AxiosError;
+      openSnackBar({ type: 'error', message: (err.response?.data as ErrorResponse).message });
```

Comment:
```suggestion
    onError: (e) => {
      if (e instanceof AxiosError) {
      openSnackBar({ type: 'error', message: err.response?.data.message });
      }
```
민재님이 남겨주신 리뷰 보면 재밌는 내용들이 많아서 한번 보면 좋을 거 같아요!
https://github.com/BCSDLab/JJBAKSA_FRONT_END/pull/262#discussion_r1483123267

Reply:
오.. 리뷰보고 감탄했습니다ㅋㅋㅋㅋㅋㅋ
앞으로도 타입단언은 최대한 사용할 일이 없도록 코드를 짜봐야할 것 같네요. c2f9f495b31d7788f055a3800534b65662e63033